hvpy/facade.py

```diff
@@ -166,8 +166,8 @@ def getStatus() -> Union[bytes, str, Dict[str, Any]]:
     Examples
     --------
     >>> from hvpy import getStatus
-    >>> getStatus()
-    {'AIA': ..., 'COSMO': ..., 'HMI': ..., 'LASCO': ..., 'SECCHI': ..., 'SWAP': ..., 'XRT': ...}
+    >>> getStatus() # doctest: +SKIP
```

Comment:
Its a shame it keeps changing, I wonder if we want to sort the return in future?

Reply:
Even sorted it would break again when a new source is added.
Does this support partial objects? Could I have it check:
```
{'AIA': ..., ...}
```
That would verify that at least something is returned.

Reply:
That should work.

Reply:
Looks like it worked a treat. Lets hope no one launches ABA soon.